package.json

```diff
@@ -34,7 +34,6 @@
     "workspace-schematic": "nx workspace-schematic",
     "dep-graph": "nx dep-graph",
     "help": "nx help",
-    "storybook:island-ui": "nx run island-ui-storybook:storybook",
```

Comment:
Why is this being removed?

Reply:
I renamed the script inside the workspace directly to bind with the "serve" script, so we can do `yarn start island-ui-storybook`